radis/io/hitemp.py

```diff
@@ -76,7 +77,10 @@ def fetch_hitemp(
 
     parallel: bool
         if ``True``, uses joblib.parallel to load database with multiple processes
-
+    database: ``str``
+        The name of the database, e.g., ``full`` or ``2010`` for CO.
+        The full option retrieves the most recent available version, while ``2010``
+        specifically selects the 2010 version. If not provided, it defaults to ``full``
```

Comment:
Update with "most_recent".
Also, it would be great if 

- "2019" would work for CO
- "2020" would work for CH4
- "2024" would work for CO2
- etc.

See the years of each release on https://hitran.org/hitemp/